assets/js/index.js

```diff
@@ -7,3 +7,72 @@ var formFocus = toolkit.formFocus;
 
 helpers.documentReady(progressiveReveal);
 helpers.documentReady(formFocus);
+
+var $ = require('jquery');
+var typeahead = require('typeahead-aria');
+var Bloodhound = require('typeahead-aria').Bloodhound;
+
+typeahead.loadjQueryPlugin();
+
+$('.typeahead').each(function applyTypeahead() {
+  var $el = $(this);
+  var $parent = $el.parent();
+  var attributes = $el.prop('attributes');
+  var $input = $('<input/>');
+  var selectedValue = $el.val();
+  var typeaheadList = $el.find('option').map(function mapOptions() {
```

Comment:
I do agree this should be the way forward though - consistency between front-end and back end code is important, and the code is already put through a transform step so adding babelify doesn't seem like too much of an overhead to me